src/sql/statement.rs

```diff
@@ -1169,7 +1185,48 @@ fn handle_create_source(scx: &StatementContext, stmt: Statement) -> Result<Plan,
                 }
             };
 
-            let mut desc = encoding.desc(envelope)?;
+            let envelope = match &envelope {
+                sql_parser::ast::Envelope::None => dataflow_types::Envelope::None,
+                sql_parser::ast::Envelope::Debezium => dataflow_types::Envelope::Debezium,
+                sql_parser::ast::Envelope::Upsert(key_format) => match connector {
+                    Connector::Kafka { .. } => {
+                        let mut key_encoding = if key_format.is_some() {
+                            get_encoding(key_format, std::collections::HashMap::new())?
```

Comment:
Can you throw a `use std::collections::HashMap` up at the top of the file so you can just write `HashMap::new()` here?